### ISSUE TITLE
Field breaks vertical stepper animation

### ISSUE THREAD

**Issue by ToxicSalt** [erikras/redux-form-material-ui#5]
When used in a vertical stepper, Field breaks the transition animation between steps. The animation should be:
current step closes upwards
next step opens downwards

what happens:
next step opens downwards
current step closes immediately

Any Field component inside a StepContent component seems to break the animation.

**Comment by erikras** [comment]
You'll need to provide an example, both with it working without `redux-form`, and with it not working _with_ `redux-form`. It seems unlikely that `redux-form` would be causing the problem.

**Comment by ToxicSalt** [comment]
@erikras 
Here is a heroku showing the issue: https://still-earth-19495.herokuapp.com/ 
git: https://github.com/ToxicSalt/client-intake-form/tree/redux-form-material-ui

It should behave like the vertical non-linear stepper here: http://www.material-ui.com/#/components/stepper

**Comment by erikras** [comment]
That's pretty weird. I tried to run it locally, but I can't get your webapp to run.

```
let router = express.Router();
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:54:16)
    at Module._compile (module.js:375:25)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/Users/erik/.Trash/client-intake-form-master/server/index.js:2:1)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
```

**Comment by DylanSimowitz** [comment]
I'm on node 6.2.2 (latest). Most likely it's your version of node.

**Comment by ToxicSalt** [comment]
Fixed in material-ui 0.15.2

**Comment by erikras** [comment]
🎉 Yay! Down to zero open issues! 👍 Thanks for reporting back.
